microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -59,57 +59,55 @@ With [Progressive Web Apps][MDNApps] (or simply *PWA*s) you do not have to decid
     :::column-end:::
 :::row-end:::
 
-By building or converting your existing site to a PWA, you can better engage your existing audience with push notifications, app like integration and offline support.  At the same time, you can continue building your audience on the open web, as users discover your PWA through search and link-sharing.  Best of all, you can update your app by simply updating your web server code.  
+By building or converting your existing site to a PWA, to better engage your existing audience with push notifications, app-like integration and offline support.  At the same time, continue to build your audience on the open web, as users discover your PWA through search and link-sharing.  Best of all, you may update your app by simply updating your web server code.  
```

Comment:
The first sentence has grammar issues

Reply:
Updated.
"By building or converting your existing site to a PWA, you engage better with your existing audience using push notifications, app-like integration, and offline support."